src/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+const api = {
+  getPulls (year) {
+    return axios.get(`http://localhost:8001/api/v1/pulls/${year}`)
```

Comment:
Sure!

Reply:
Maybe, we should have an issue for making sure that there is no hard-coded stuff in the whole project. I think it will not be a very difficult task considering the size of the project.

Reply:
That is kinda part of #10, but we can create more issues for clarification.